tests/appsec/iast/sink/test_insecure_cookie.py

```diff
@@ -11,8 +11,20 @@
 
 
 @coverage.basic
-@released(dotnet="?", java="?", golang="?", php_appsec="?", python="?", ruby="?")
+@released(dotnet="?", golang="?", php_appsec="?", python="?", ruby="?")
 @released(nodejs={"express4": "4.1.0", "*": "?"})
+@released(
```

Comment:
When the release version is the same, it's probably better to just do `java="1.18.0"` and then use additional `@missing_feature(weblog_variant="akka-http", reason="Missing endpoint")` to exclude some variants.

Reply:
Indeed, and you can even do : 

```python
@released(java={"akka-http": "?", "*": "1.18.0"})
```